res/css/photon-components.css

```diff
@@ -79,3 +93,40 @@
   border: 1px solid var(--red-60);
   box-shadow: 0 0 0 1px var(--red-60), 0 0 0 4px rgba(251, 0, 34, 0.3);
 }
+
+/* --- Message bars --- */
+
+/* See https://design.firefox.com/photon/components/message-bars.html */
+.photon-message-bar {
+  display: flex;
+  width: 100%;
+  min-height: 32px;
+  box-sizing: border-box;
+  align-items: center;
+  padding: 0 4px;
+  border-radius: 4px;
+  font-size: 13px;
+  font-weight: 400;
+  line-height: 1.4;
+}
+
+.photon-message-bar-error {
+  /* Note: 32px is: 4px (left padding for the message bar) + 4px (left padding
+   * for the icon) + 16px (icon width) + 4px (right padding for the icon) + 4px
+   * (space between icon and text) */
+  padding: 4px 4px 4px 32px;
+
+  /* Note: 8px is: 4px (left padding for the message bar) + 4px (left padding
+   * for the icon) */
+  background: url(../img/svg/error.svg) no-repeat 8px center / 16px 16px
+    var(--red-60);
```

Comment:
over the night I thought that this `background` could go to  `.photon-message-bar` and we'd change just the `background-image` and `background-color` here. But let's think of this when we implement more types of message bars.